main.js

```diff
@@ -90,6 +89,48 @@ const renderMainWindow = () => {
         });
     }
   })();
+  
+  //read file and show it in app
+  ipcMain.handle('request-load-text', () => {
+    return fs.readFileSync(filePath, "utf8");
```

Comment:
I think, it's better to change this string to const and add const to up of this file.
```
const ENCODING = 'utf8'
```